### DISCUSSION TITLE
v1.0.0-alpha

### DISCUSSION THREAD

**Discussion by myckhel** [binkode/laravel-chat-system#6] [Ideas]
## Features
- Message Broadcasting Events
- Message State Events
- Conversation Support
- Multiple Conversation Users
- Integrated Job Queues
- Customisable
*...more*

<hr /><em>This discussion was created from the release <a href='https://github.com/myckhel/laravel-chat-system/releases/tag/v1.0.0-alpha'>v1.0.0-alpha</a>.</em>

**Comment by hirbod** [comment]
Message types (text, video, audio, location) and mixed (image + text)

Events when both parties have deleted the chat (for example to delete stored images).

**Reply by myckhel** [reply]
Nice Idea 💡 

> Message types (text, video, audio, location) and mixed (image + text)

Scheduled to be implemented after the basic stable release: https://github.com/myckhel/laravel-chat-system/projects/3

> Events when both parties have deleted the chat (for example to delete stored images).

Just created a task and will be worked on.
https://github.com/myckhel/laravel-chat-system/projects/2#card-61840482
